packages/ui/src/domainModules/collectionMammals/components/RecordNavigationBar/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Button, Icon, Grid } from 'semantic-ui-react'
+
+export const RecordNavigationBar = ({
+  recordNumber,
+  totalRecords,
+  onPreviousRecord,
+  onNextRecord,
+  onShowAllRecords,
+  onNewRecord,
+}) => {
+  return (
+    <Grid textAlign="center">
+      <Grid.Column width={2}>
+        <Button
+          icon
+          onClick={() => onPreviousRecord({ recordNumber })}
+          disabled={recordNumber === 1}
+        >
+          <Icon name="chevron left" />
+        </Button>
+        <Button
+          icon
+          onClick={() => onNextRecord({ recordNumber })}
+          disabled={recordNumber === totalRecords}
+        >
+          <Icon name="chevron right" />
+        </Button>
+      </Grid.Column>
+      <Grid.Column width={2}>
+        {totalRecords}
+        <br />Total records
+      </Grid.Column>
+      <Grid.Column width={1}>
+        <Button icon onClick={() => onShowAllRecords()}>
+          <Icon name="book" />
+        </Button>
+      </Grid.Column>
+      <Grid.Column width={1}>
+        <Button icon onClick={() => onNewRecord()}>
+          <Icon name="plus" />
+        </Button>
+      </Grid.Column>
+      <Grid.Column width={9} />
+      <Grid.Column width={1}>
+        <Button icon onClick={() => onToggleFilters()}>
+          <Icon name="search" />
+        </Button>
+      </Grid.Column>
+    </Grid>
+  )
+}
+
+// RecordNavigationBar.propTypes = {
+//   recordNumber: PropTypes.number.isRequired,
+//   onNextRecord: PropTypes.func,
```

Comment:
we define propTypes above the component, you can see the ordering convention here: https://github.com/airbnb/javascript/tree/master/react#ordering

for the handler functions you can use the `oneOfType` PropType (https://reactjs.org/docs/typechecking-with-proptypes.html) to allow it to be either a `func` or `bool`, because if we do not pass a handler we can pass false.